### PR TITLE
Seeder

### PR DESCRIPTION
**Pull request by Banovvv**

### CODE REVIEW

File: SportsHub/SportsHub.DAL/SportsHub.DAL.csproj [hunk 1]
```diff
@@ -17,6 +17,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```
Comment (MBocharskyi) [inline SportsHub/SportsHub.DAL/SportsHub.DAL.csproj:20]: Why do you need this package? Is it possible to use System.Text.Json?

File: SportsHub/SportsHub.DAL/Seeding/Seeder.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        {
+            IEnumerable<User> users = JsonConvert.DeserializeObject<List<User>>(_users);
+
+            context.Users.AddRange(users);
+            await context.SaveChangesAsync();
+        }
+
+        private async Task SeedRolesAsync(ApplicationDbContext context)
+        {
+            IEnumerable<Role?> roles = JsonConvert.DeserializeObject<List<Role?>>(_roles);
```
Comment (MBocharskyi) [inline SportsHub/SportsHub.DAL/Seeding/Seeder.cs:30]: Why do you use nullable for Role?

File: SportsHub/SportsHub.DAL/Seeding/Seeder.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
+﻿using Newtonsoft.Json;
+using SportsHub.DAL.Data;
+using SportsHub.Domain.Models;
+
+namespace SportsHub.DAL.Seeding
+{
+    public class Seeder : ISeeder
+    {
+        #region Raw Data
+        private readonly string _roles = "[{\"Name\":\"User\"},{\"Name\":\"Admin\"}]";
```
Comment (MBocharskyi) [inline SportsHub/SportsHub.DAL/Seeding/Seeder.cs:10]: Why do you use json? Are you sure that it has correct format? Did you consider creating data generation method?

File: SportsHub/SportsHub.DAL/SportsHub.DAL.csproj (hunk 1 above)
Comment (Banovvv) [reply SportsHub/SportsHub.DAL/SportsHub.DAL.csproj:20]: Updated to use System.Text.Json.

File: SportsHub/SportsHub.DAL/Seeding/Seeder.cs (hunk 2 above)
Comment (Banovvv) [reply SportsHub/SportsHub.DAL/Seeding/Seeder.cs:30]: It was by mistake, removed.

**Review by nakometa — APPROVED** [review]

**Pull request merged**